src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -435,6 +437,34 @@ public String getReferenceJobName() {
         return referenceJobName;
     }
 
+    /**
+     * Sets the reference build id to get the results for the issue difference computatation.
+     *
+     * @param referenceBuildId
+     *         the build id of the reference job
+     */
+    @DataBoundSetter
```

Comment:
Remove `@DataBoundSetter`: I'm not sure if this is correct but I think that a Freestyle Build will never be configured in that way, your new option makes sense only for pipelines where you are computing the build ID on your own.

Reply:
I only added this because all other configuration options had it.  It probably does not make sense outside of a pipeline, at least unless support is added for a more dynamic UI based build id selector.  
I've removed this.